runtime/near-test-contracts/congestion-control-test-contract/src/lib.rs

```diff
@@ -0,0 +1,204 @@
+//! This contract is used for congestion control tests that are sensitive to contract size.
+//! For some tests large contract size causes less congestion, and the checks fail.
+//! Let's use a separate small contract for these tests.
+
+#![allow(clippy::all)]
+
+#[allow(unused)]
+extern "C" {
```

Comment:
1) Is all of it needed?
2) Is there any way we can reuse the existing one or refactor it to avoid copy pasting?